UncertainSCI/utils/compute_mom.py

```diff
@@ -0,0 +1,63 @@
+import numpy as np
+import scipy.special as sp
+
+from UncertainSCI.utils.compute_subintervals import compute_subintervals
+from UncertainSCI.utils.quad import gq_modification_composite
+from UncertainSCI.utils.quad import gq_modification_unbounded_composite
+
+def compute_mom_bounded(a, b, weight, k, singularity_list, Nquad=10):
```

Comment:
Are these functions for computing moments? (As mentioned in one of the doc comments). If so, I'd prefer the function name spelled that out, the goofy abbreviation "mom" is not worth the saved 3-4 characters.